jbpm-designer-client/src/main/resources/org/jbpm/designer/public/js/Plugins/shapemenu.js

```diff
@@ -311,6 +324,28 @@ ORYX.Plugins.ShapeMenuPlugin = {
             });
 		}
 	},
+
+	showElementInDocs: function() {
+		var showInDocs = true;
+		if(this.currentShapes[0] instanceof ORYX.Core.Edge) {
+			// condition under which edge is displayed in the generated docs
+			if(!this.currentShapes[0].properties['oryx-name'].length && !this.currentShapes[0].properties['oryx-documentation'].length && !this.currentShapes[0].properties['oryx-conditionexpression'].length) {
+				showInDocs = false;
+			}
+		}
+		if(showInDocs) {
+			ORYX.PROCESSDOC_RESOURCEID = this.currentShapes[0].resourceId;
+			Ext.getCmp('maintabs').setActiveTab(2);
+		} else {
+			this.facade.raiseEvent({
+				type 		: ORYX.CONFIG.EVENT_NOTIFICATION_SHOW,
+				ntype		: 'info',
+				msg         : ORYX.I18N.ShapeMenuPlugin.unableToFindNodeSource,
```

Comment:
Should this be  ORYX.I18N.ShapeMenuPlugin.unableToLinkElementToDocs ?
For this constant, maybe "Element not included in documentation" might be better, as the current message "Unable to..." makes it sound as though there's an error.